doc/rst/source/explain_symbols.rst_

```diff
@@ -1,29 +1,28 @@
 **-S**\ [*symbol*][*size*]
     Plot symbols (including vectors, wedges, fronts, decorated or quoted lines).
-    If present, *size* is symbol size in the unit set in
-    :doc:`gmt.conf` (unless **c**, **i**, or **p** is appended). If the symbol
+    If present, *size* is symbol size in the default unit set by
+    :term:`PROJ_LENGTH_UNIT` (unless **c**, **i**, or **p** is appended). If the symbol
     code (see below) is not given it will be read from the last column in
-    the input data; this cannot be used in conjunction with binary input.
-    Optionally, append **c**, **i**, or
-    **p** to indicate that the size information in the input data is in
-    units of cm, inch, or point, respectively [Default is
+    the input data; this scheme cannot be used in conjunction with binary input.
+    Optionally, append **c**, **i**, or **p** to indicate that the size information
+    in the input data is in units of cm, inch, or point, respectively [Default is
     :term:`PROJ_LENGTH_UNIT`]. **Note**: If you provide *both* size and symbol via the
-    input file you must use :term:`PROJ_LENGTH_UNIT` to indicate the unit
-    used for the symbol size or append the units to the sizes in the file.
-    If symbol sizes are expected via the third data column then you may convert
+    input file you *must* use :term:`PROJ_LENGTH_UNIT` to indicate the unit
+    used for the symbol size or append the units to the sizes listed in the file.
+    If symbol sizes are expected via the a data column then you may convert
     those values to suitable symbol sizes via the **-i** mechanism. The general
     input expectations are:
 
-    *x y* [ *z* ] [ *size* ] [ *symbol-parameters* ] [ *symbol* ]
+    *coordinates* [ *value* ] [ *size* ] [ *symbol-parameters* ] [ *symbol* ]
 
-    where the optional *z* is required when **-C** is used, the optional
+    where the optional *value* is required when **-C** is used, the optional
     *size* is required when no symbol size is specified, and the trailing
```

Comment:
It's still unclear how to specify *size* in the input file for symbols with more than one size parameters. For example, for a rectangle, users may think that the *size* should be `width/height` or `width height`.

Reply:
Will look.

Reply:
That is what [ *size* ] [ *symbol-parameters* ] means, no? If there is more than one size argument then that means you have symbol-parameters.

Reply:
It's a little confusing for me, especially for some symbols like -Se (-Sedirection/major_axis/minor_axis, what does *size* means?)

Reply:
Maybe part of the problem is the listing of two things: [ *size* ] [ *symbol-parameters* ].  What if it was just *size* and we say the one or more columns needed to supply the size parameter(s)?

Reply:
For **-Sb**, *base* can also be given in the input file, is *base* part of *size* or *symbol-parameters*?
```
-Sb[size[c|i|p|u]][+b|B[base]]
```

Reply:
Hm, no, I would say that is part of _symbol-parameters_.  Maybe we need to say that *size* can be more than one column depending on the symbol, (e.g., _width_ and _height_ for a rectangle requires two columns) while some symbol has additional parameters (such as _azimuth_ for an oriented symbol).  Would that be clearer?  Actually, I think that fails too because the ellipse starts with teh angle, then the two dimensions, so listing size first fails.  I think the bets would be a single entry called symbol-parameters which may be one or more parameters needed to define the symbol, such as diameter for a circle, width and height for a rectangle, and direction, major and minor axies for an ellipse.

Reply:
> I think the bets would be a single entry called symbol-parameters which may be one or more parameters needed to define the symbol, such as diameter for a circle, width and height for a rectangle, and direction, major and minor axies for an ellipse.

It sounds good to me, but I didn't check if it works for other symbols (some symbols I never used before).